packages/web3-shared/src/types/index.ts

```diff
@@ -124,6 +124,9 @@ export interface ERC721TokenInfo {
 export interface ERC721TokenDetailed {
     tokenId: string
     info: ERC721TokenInfo
+    address?: string
+    name?: string
+    logoURI?: string
```

Comment:
Remove these there properties, since they are duplicated  with `contractDetailed['address']`, `info['name']` and `info['image']`.